src/components/accession2/edit/EditState.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
```

Comment:
this was extracted out from EditStateModal... and it now also instantiated in quantity modal when we edit Used Up status.